### PR TITLE
Add PyQt5-sip (required for PyQt5 >= 5.11)

### PR DESCRIPTION
**Pull request by cbrnr**
Let's see if this fixes CircleCI for good.

### CODE REVIEW

**Comment by cbrnr** [comment]
Nope. Now we get

```
#!/bin/bash -eo pipefail
python -c "from mayavi import mlab; import matplotlib.pyplot as plt; mlab.figure(); plt.figure()"
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-circleci'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-circleci'
libgcc_s.so.1 must be installed for pthread_cancel to work
Received 'aborted' signal
```

Is this related to https://github.com/ContinuumIO/anaconda-issues/issues/9190 even though we install via `pip`?

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
               - data-cache
 
         # Get latest Anaconda running
+        - run: sudo apt-get install libxss1
```
Comment (jasmainak) [inline .circleci/config.yml]: I guess we'll also have to update the docs ... ?

Comment (larsoner) [reply .circleci/config.yml]: Based on the symptoms and related issue, I suspect that this is a container-specific problem that users won't really see. Travis does not have this problem, and the multiple users I know that run `environment.yml` have not hit this problem on their computers.

**Comment by agramfort** [comment]
don't know. @larsoner should know

**Comment by larsoner** [comment]
It is probably overkill to use `--no-deps`. IIRC in the past at least there were problems with just plain `--upgrade` with `pip`, like it would upgrade all packages in the `requirements` even if what was currently installed met them. IIRC doing `--no-deps` will only upgrade packages that must be updated to fulfill requirements.

**Comment by cbrnr** [comment]
Can you rule out that this is not causing the issue? Maybe it is safer to use `pip install --upgrade --upgrade-strategy only-if-needed PyQt5 PyQt5-sip`?

**Comment by larsoner** [comment]
Feel free to try it, that would be an easy fix

**Comment by cbrnr** [comment]
OK, and we can even omit the `--upgrade-strategy only-if-needed` part because this is the default in the latest version of `pip`.

**Comment by cbrnr** [comment]
Doesn't work, this would have been too easy. But I think we can still remove the argument.

**Comment by larsoner** [comment]
We can remove the upgrade line altogether, the only reason we had it was to avoid the error we now get anyway

**Comment by cbrnr** [comment]
Does `PyQt5` get installed before that line?

**Comment by larsoner** [comment]
Circle worked!

Now I'll try to remove the `apt-get` lines to see if they were indeed necessary.

**Comment by cbrnr** [comment]
I bet they aren't necessary anymore. And it's good to see that a temporary hack is not necessary anymore.

**Comment by cbrnr** [comment]
Uh oh...

**Comment by larsoner** [comment]
This is useful! It means that the libs present on the system *do* cause this failure. So if we see it again, we need to hunt them down. And that linked thread tells us the commands to do it.

**Comment by larsoner** [comment]
Thanks @cbrnr !

**Pull request merged**